rules/ntriplify.py

```diff
@@ -0,0 +1,11 @@
+### A simple script for converting all Turtle RDF files in the current directory to the N-triples RDF format
+
+import os
```

Comment:
Is it possible to specify the list of files from the command line. You may use argparse for parsing the list of RDF files. You can also specify an option for a directory (containing the list of RDF files).

Reply:
added [here](https://github.com/VCityTeam/UD-Graph/blob/aa846ff115fd7ce7eb5a3348a11464d4a3b35351/rules/ntriplify.py#L5)
created an [issue](https://github.com/VCityTeam/UD-Graph/issues/32) for directory option.